lms/djangoapps/bulk_email/models.py

```diff
@@ -115,6 +123,12 @@ def get_users(self, course_id, user_id=None):
             return use_read_replica_if_available(enrollment_qset.exclude(id__in=staff_instructor_qset))
         elif self.target_type == SEND_TO_COHORT:
             return self.cohorttarget.cohort.users.filter(id__in=enrollment_qset)  # pylint: disable=no-member
+        elif self.target_type == SEND_TO_TRACK:
+            return use_read_replica_if_available(
+                enrollment_qset.filter(
+                    courseenrollment__mode=self.coursemodetarget.track.mode_slug
+                )
+            )
```

Comment:
@schenedx Can you (or someone else on ECOM) confirm that this logic will correctly fetch all users enrolled in a given mode? I'm essentially grabbing all entries in the `CourseEnrollment` table [model link](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/models.py#L969) that match the given `course_id` and `mode`, with `is_active` set to True.